docker-compose.yml

```diff
@@ -1,300 +1,198 @@
+    ports:
+      - "6379:6379"
+    command: redis-server --requirepass ${REDIS_PASSWORD}
+    
   tika:
-    image: apache/tika:2.9.1.0-full
-    container_name: tika
+    image: trieve/tika:latest
     healthcheck:
-      test:
+      test: 
         [
           "CMD-SHELL",
           "wget --spider -S 'http://tika:9998/version' 2>&1 | grep 'HTTP/' | awk '{print $2}' | grep 200 || exit 1",
         ]
       interval: 10s
       timeout: 5s
       retries: 10
-    networks:
-      - app-network
     ports:
       - 9998:9998
-    profiles: [dev]
 
-  server:
-    image: trieve/server:no-ocr
-    container_name: server
-    build:
-      context: ./server/
-      dockerfile: Dockerfile.no-ocr
-    depends_on:
-      tika:
-        condition: service_healthy
-      db:
-        condition: service_healthy
-      qdrant-database:
-        condition: service_started
-      redis:
-        condition: service_healthy
-      keycloak:
-        condition: service_healthy
-    networks:
-      - app-network
+  qdrant-database:
+    image: trieve/qdrant:latest
+    environment:
+      QDRANT__SERVICE__API_KEY: &qdrant_api_key ${QDRANT_API_KEY}
     ports:
-      - 8090:8090
+      - 6333:6333
+      - 6334:6334
+    
+  splade-embeddings:
+    image: trieve/splade-embeddings:latest
+    ports:
+      - 7070:7070
     environment:
-      - ADMIN_API_KEY=${ADMIN_API_KEY}
-      - BASE_SERVER_URL=${BASE_SERVER_URL}
-      - REDIS_URL=${REDIS_URL}
-      - QDRANT_URL=${QDRANT_URL}
-      - QDRANT_API_KEY=${QDRANT_API_KEY}
-      - DATABASE_URL=${DATABASE_URL}
-      - SMTP_RELAY=${SMTP_RELAY}
-      - SMTP_USERNAME=${SMTP_USERNAME}
-      - SMTP_PASSWORD=${SMTP_PASSWORD}
-      - SMTP_EMAIL_ADDRESS=${SMTP_EMAIL_ADDRESS}
-      - OPENAI_API_KEY=${OPENAI_API_KEY}
-      - LLM_API_KEY=${LLM_API_KEY}
-      - SECRET_KEY=${SECRET_KEY}
-      - SALT=${SALT}
-      - S3_ENDPOINT=${S3_ENDPOINT}
-      - S3_ACCESS_KEY=${S3_ACCESS_KEY}
-      - S3_SECRET_KEY=${S3_SECRET_KEY}
-      - S3_BUCKET=${S3_BUCKET}
-      - COOKIE_SECURE=${COOKIE_SECURE}
-      - QDRANT_COLLECTION=${QDRANT_COLLECTION}
-      - TIKA_URL=${TIKA_URL}
-      - OPENAI_BASE_URL=${OPENAI_BASE_URL}
-      - STRIPE_SECRET=${STRIPE_SECRET}
-      - STRIPE_WEBHOOK_SECRET=${STRIPE_WEBHOOK_SECRET}
-      - ADMIN_DASHBOARD_URL=${ADMIN_DASHBOARD_URL}
-      - OIDC_CLIENT_SECRET=${OIDC_CLIENT_SECRET}
-      - OIDC_CLIENT_ID=${OIDC_CLIENT_ID}
-      - OIDC_AUTH_REDIRECT_URL=${OIDC_AUTH_REDIRECT_URL}
-      - OIDC_ISSUER_URL=${OIDC_ISSUER_URL}
-      - GPU_SERVER_ORIGIN=${GPU_SERVER_ORIGIN}
-      - AWS_REGION=${AWS_REGION}
-      - SPARSE_SERVER_ORIGIN=${SPARSE_SERVER_ORIGIN}
-      - EMBEDDING_SERVER_ORIGIN=${EMBEDDING_SERVER_ORIGIN}
-      - RERANKER_SERVER_ORIGIN=${RERANKER_SERVER_ORIGIN}
-      - UNLIMITED=${UNLIMITED}
-      - REDIS_CONNECTIONS=${REDIS_CONNECTIONS}
+      - HF_TOKEN=${HF_TOKEN}
+      - API_KEY=${OPENAI_API_KEY}
+    env_file:
+      .env
```

Comment:
The splade embeddings and ingestion-service should be expanded out instead of reading .env

Reply:
Ingestion-service can be the exact same as the server, just annoying to have to double the size of the docker-compose.yml file

Reply:
Splade-embeddings only needs 
```
HF_TOKEN
API_KEY
and SENTRY_URL
```